server/package.json

```diff
@@ -19,7 +19,7 @@
     "@sentry/node": "^6.2.5",
     "@sentry/tracing": "^6.2.5",
     "axios": "^0.21.2",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v9.0.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-pa-tweaks",
```

Comment:
Reminder to update this after https://github.com/bravetechnologycoop/brave-alert-lib/pull/49 is merged

Reply:
Done